ush/config_defaults.yaml

```diff
@@ -1797,111 +1661,156 @@ task_make_lbcs:
+      global_albedo4.1x1.grb: '{{ workflow.FIXam }}/global_albedo4.1x1.grb'
+      global_zorclim.1x1.grb: '{{ workflow.FIXam }}/global_zorclim.1x1.grb'
+      global_tg3clim.2.6x1.5.grb: '{{ workflow.FIXam }}/global_tg3clim.2.6x1.5.grb'
+      sfc_emissivity_idx.txt: '{{ workflow.FIXam }}/global_sfc_emissivity_idx.txt'
+      solarconstant_noaa_an.txt: '{{ workflow.FIXam }}/global_solarconstant_noaa_an.txt'
+      global_o3prdlos.f77: '{{ workflow.FIXam }}/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77'
+      fd_ufs.yaml: '{{ user.UFS_WTHR_MDL_DIR }}/tests/parm/fd_ufs.yaml'
+      INPUT/grid_spec.nc: '{{ task_make_grid.GRID_DIR }}/{{ "CRES" | env }}_mosaic.halo{{ constants.NH3 }}.nc'
+      INPUT/grid.tile7.halo4.nc: '{{ task_make_grid.GRID_DIR }}/{{ "CRES" | env }}_grid.tile7.halo{{ constants.NH4 }}.nc'
+      INPUT/{{ "CRES" | env }}_grid.tile7.halo{{ constants.NH3 }}.nc: '{{ task_make_grid.GRID_DIR }}/{{ "CRES" | env }}_grid.tile7.halo{{ constants.NH3 }}.nc'
+      INPUT/oro_data.nc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}_oro_data.tile7.halo{{ constants.NH0 }}.nc'
+      INPUT/oro_data.tile7.halo{{ constants.NH4 }}.nc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}_oro_data.tile7.halo{{ constants.NH4 }}.nc'
+    lateral_boundary_conditions:
+      interval_hours: !int '{{ task_get_extrn_lbcs.envvars.LBC_SPEC_INTVL_HRS }}'
+      offset: 0
+      path: '{{ task_run_fcst.fv3.rundir }}/INPUT/{{ nco.NET_default }}.t{{ timevars.hh }}z.gfs_bndy.tile{tile}.f{forecast_hour:03d}.nc'
+    length: !int '{{ workflow.FCST_LEN_HRS }}'
+    model_configure:
+      update_values:
+        start_year: !int '{{ cycle.year }}'
+        start_month: !int '{{ cycle.month }}'
+        start_day: !int '{{ cycle.day }}'
+        start_hour: !int '{{ cycle.hour }}'
+        start_minute: !int '{{ cycle.minute }}'
+        start_second: !int '{{ cycle.second }}'
+        nhours_fcst: !int '{{ workflow.FCST_LEN_HRS }}'
+        fhrot: 0
+        run_continue: false
+        ens_sps: false
+        dt_atmos: !int '{{ task_run_fcst.DT_ATMOS // 1 }}'
+        calendar: julian
+        memuse_verbose: false
+        restart_interval: 0
+        output_1st_tstep_rst: false
+        write_dopost: false
+        ideflate: 0
+        nbits: 0
+        ichunk2d: -1
+        jchunk2d: -1
+        ichunk3d: -1
+        jchunk3d: -1
+        kchunk3d: -1
+        itasks: 1
+        quilting: true
+        write_groups: !int '{{ task_run_fcst.WRTCMP_write_groups // 1 }}'
+        write_tasks_per_group: !int '{{ task_run_fcst.WRTCMP_write_tasks_per_group // 1 }}'
+        num_files: 2
+        filename_base: 'dyn phy'
+        output_file: 'netcdf netcdf'
+        output_fh: '1  -1'
+        nsout: -1
+        output_grid: '{{ task_run_fcst.WRTCMP_output_grid }}'
+    namelist:
+      base_file: '{{ workflow.EXPTDIR }}/input.nml.{{ workflow.CCPP_PHYS_SUITE }}'
+      update_values:
+        atmos_model_nml:
+          blocksize: !int '{{ task_run_fcst.BLOCKSIZE // 1 }}'
```

Comment:
Sorry I missed this comment earlier.

It's meant to avoid a problem when there are multiple steps to render values. In a small example:

```
a: 2
b: 7
foo: !int "{{ a * b }}"
bar: !int "{{ foo }}"
``` 

Realizing this config:

```
$ uw config realize -i example.yaml --output-format yaml
a: 2
b: 7
foo: 14
bar: !int '!int 14'
```

Adding the `// 1` to `bar: !int "{{ foo // 1 }}"`  gives the intended `bar: 14` result. I believe this allows the `bar` value to hold off until `foo` has an actual integer value. The integer divide by 1 raises an exception except for when the numerator is actually a number, and our renderer allows for broad exceptions that result in the expression _not_ being rendered. 

It's a cheeky workaround to have the values wait until it's appropriate to render the expression.

Reply:
IMO it would be better if we could deal with the core issue in `uwtools` instead of using this workaround, whose meaning may not be clear to users/maintainers. It's pretty subtle. And there are now a lot of instances of it... Don't wait for me, but let me see if there's a way we could address this correctly. Thanks for the minimal reproducer!